### PR TITLE
Audio-only calls

### PR DESCRIPTION
**Pull request by or-else**
Calls with Android and web-web work, nothing crashes, sound comes through. Switching back to video call does not work.

### CODE REVIEW

File: src/widgets/call-panel.jsx [hunk 1]
```diff
@@ -410,26 +412,28 @@ class CallPanel extends React.PureComponent {
 
   render() {
     const remoteActive = this.remoteRef.current && this.remoteRef.current.srcObject;
-    const disabled = !(this.state.localStream && this.state.localStream.getTracks());
-    const audioIcon = this.state.localStream && this.state.localStream.getAudioTracks()[0].enabled ? 'mic' : 'mic_off';
-    const videoIcon = this.state.localStream && this.state.localStream.getVideoTracks()[0].enabled ? 'videocam' : 'videocam_off';
+    const audioTracks = this.state.localStream && this.state.localStream.getAudioTracks();
+    const videoTracks = !this.state.audioOnly && this.state.localStream && this.state.localStream.getVideoTracks();
+    const disabled = !(audioTracks && audioTracks[0]);
+    const audioIcon = audioTracks && audioTracks[0] && audioTracks[0].enabled ? 'mic' : 'mic_off';
```
Comment (aforge) [inline src/widgets/call-panel.jsx:418]: Will `audioTracks[0]` cause an error if `audioTracks` is an empty array?

Comment (or-else) [reply src/widgets/call-panel.jsx:418]:
```javascript
let x = [];
console.log(x);
```

The output is `undefined`. No crash.

**Comment by or-else** [comment]
The issue with FF immediately hanging up is due to Android interference: there is a bug in Android: it incorrectly assumes it's in a call and because of that it declines all calls.

**Comment by or-else** [comment]
The issue with app not ringing due to no data fetch is fixed: https://github.com/tinode/webapp/pull/108/commits/2d76befea1b444656c8dc0206806414e18df159e

**Comment by or-else** [comment]
Do you want to take another look? Or should I merge this?

**Pull request merged**